Update/omot_025.txt

```diff
@@ -1983,7 +1983,7 @@ void main()
 //rうつむき加減で踏む場所を誤ったのか、梨花ちゃんは浮石に足をとられて躓いて倒れる。そして膝をぶつけたのか、その場に顔をしかめてうずくまった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "うつむき加減で踏む場所を誤ったのか、梨花ちゃんは浮石に足をとられて躓いて倒れる。そして膝をぶつけたのか、その場に顔をしかめてうずくまった。",
-			NULL, "She must've misstepped while looking down, because Rika-chan stumbled on a rock and fell to the ground, then hit her knee on something and crouched down on the spot in pain.", Line_Normal);
+			NULL, "She must've misstepped while looking down, because Rika-chan stumbled on a rock and fell to the ground. She must have hit her knee on something, because her face twisted with pain as she remained on the ground.", Line_Normal);
```

Comment:
```suggestion
			NULL, "She must've misstepped while looking down, because Rika-chan stumbled on a rock and fell to the ground. And her face twisted in pain as she remained on the ground, like she'd hit her knee on something.", Line_Normal);
```